src/IMangrove.sol

```diff
@@ -40,16 +39,37 @@ interface IMangrove {
     uint id,
     uint prev
   );
-  event OrderComplete(
+  // tried to add all relevant data for an order.
+  // same
+  event OrderComplete( // this could be deduced from the OfferSuccess events emtitted from "execute" in Mangrove
     address indexed outbound_tkn,
     address indexed inbound_tkn,
     address indexed taker,
+    uint takerWants,
+    uint takerGives,
     uint takerGot,
     uint takerGave,
+    bool fillWants,
+    uint penalty,
+    uint feePaid
+  );
+  // added this event, because we used the OrderComplete event before, but this did not makes sense for Snipes, when we now also have takerWants and takerGives in OrderComplete
+  // snipes does not have 1 wants and gives, but an array of them.
+  // this could be deduced from the OfferSuccess events emtitted from "execute" in Mangrove
+  // same
+  event SnipesComplete( // this might be to much to add in the event
```

Comment:
I am not a fan of having a separate log. What do we gain besides the `targets`? 
We actually know the targets from the `OfferSuccess` or `OfferFail` logs. We could have later a frontend that cooks its own market order in the form of snipes, and you would not want to distinguish between the two logs

Reply:
Market orders and snipes are different types of orders following different rules, so IMO it makes sense to emit different events for them, so that consumers can distinguish between them.

Reply:
Then maybe just adding a field `snipe:bool` to `OrderComplete`?